Tests/Admin/AdminTest.php

```diff
@@ -846,6 +851,19 @@ public function testGetBaseCodeRoute()
         $this->assertSame('foo', $admin->getBaseCodeRoute());
     }
 
+    // NEXT_MAJOR: uncomment this method.
+    // public function testGetBaseCodeRoute()
```

Comment:
Could you please create a PR against master after this one is merged and 3.x is merged into master, so we can check if THIS test and code is working.

@greg0ire but i would say this new (master-)PR should be closed afterwards, as it would create merge conflicts in the future, am I right?